tests/lean/PPInstances.lean.expected.out

```diff
@@ -0,0 +1,6 @@
+PPInstances.lean:27:68-27:73: warning: declaration uses 'sorry'
+@Module R E _ _ : Type
+@Module R E (_ : Semiring R) (_ : AddCommMonoid E) : Type
```

Comment:
This combination `pp.instances=false, pp.instances.withType=true` is great for spotting these nested instances. Wow, I didn't even realize that the nesting could get so deep. Here is a goal from binport that I have been struggling to synthesize:
```lean
(_ :
                HasLiftT
                  (@coeSort'
                    (@Subspace Real E (_ : Field Real) (_ : AddCommGroup E)
                      (_ : @Module Real E (_ : Semiring Real) (_ : AddCommMonoid E)))
                    (_ :
                    HasCoeToSort
                      (@Subspace Real E (_ : Field Real) (_ : AddCommGroup E)
                        (_ : @Module Real E (_ : Semiring Real) (_ : AddCommMonoid E))))
                    p)
                  E)
```

Reply:
It looks good, but (minor nit) it feels a bit weird that `pp.instances.withType` does something when `pp.instances` is disabled. Maybe `pp.instanceTypes` would be a more appropriate name?